egs/wsj/s5/utils/perturb_data_dir_speed.sh

```diff
@@ -112,4 +112,9 @@ cat $srcdir/utt2dur | utils/apply_map.pl -f 1 $destdir/utt_map  | \
 
 rm $destdir/spk_map $destdir/utt_map 2>/dev/null
 echo "$0: generated speed-perturbed version of data in $srcdir, in $destdir"
-utils/validate_data_dir.sh --no-feats $destdir
+
+if [ -f $srcdir/text ]; then
```

Comment:
This pattern where we have to check things and then specify --no-text, --no-feats, is getting a bit tiresome.
I just checked validate_data_dir.sh, and it looks safe to supply any options where it might be necessary.
So just supply --no-feats --no-text here